Update screenshots

Stypox opened the pull request:
<!-- Hey there. Thank you so much for improving NewPipe, and filling out the details. Having roughly the same layout helps everyone considerably :)-->

#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
I used #9182 to take screenshots, so that they don't become outdated when we release 0.26.0. The screenshots resemble the previous screenshots, except that they are all the same size (16:9), they try to show off as much public domain material as possible, and there are no screenshots for Downloads.

All images have been stripped of metadata and optimized with both `mogrify -strip -taint -compress Lossless` and `optipng`.

You can view a preview [here](https://github.com/Stypox/NewPipe/tree/update-screenshots).

#### APK testing
<!-- Use a new, meaningfully named branch. The name is used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe, e.g. "commentfix", if your PR implements a bugfix for comments. (No names like "patch-0" and "feature-1".)  -->
<!-- Remove the following line if you directly link the APK created by the CI pipeline. Directly linking is preferred if you need to let users test.-->
The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

TobiGr reviewed (commented):
Looks good. Can you add one screenshot using the light theme on a phone, too? Maybe take one screenshot twice so the difference is clearly visible. You could also make sure to display the two tablet screenshots in the same line by adding an extra lilnebreak after the last phone screenshot.

What about the translated READMEs? Do we need to modify their image URLs, too?

Stypox commented:
@TobiGr I have changed the "phone search" screenshot to be white-themed, and the "tablet search" one to be dark-themed. I added two `<br/><br/>` between the tablet and phone images. I also updated all translated READMEs.
